frontend/src/lib-customizations/defaults/employee/i18n/fi.tsx

```diff
@@ -3076,7 +3076,8 @@ export const fi = {
       RETROACTIVE: 'Takautuva päätös',
       NO_STARTING_PLACEMENTS: 'Piilota uudet aloittavat lapset',
       MAX_FEE_ACCEPTED: 'Suostumus korkeimpaan maksuun',
-      PRESCHOOL_CLUB: 'Vain esiopetuksen kerho'
+      PRESCHOOL_CLUB: 'Vain esiopetuksen kerho',
+      NO_OPEN_INCOME_STATEMENTS: 'Piilota avonaiset tuloselvitykset'
```

Comment:
Jäin miettii mikä tähän ois kuvaavin teksti. Tässä listataan päätöksiä eli tuloselvityksiä ei piiloteta. Ehkä joku "Ei avonaisia tuloselvityksiä" vois olla linjassa noiden muiden kanssa.